opendbc/car/gm/values.py

```diff
@@ -245,7 +249,10 @@ class CanBus:
 EV_CAR = {CAR.CHEVROLET_VOLT, CAR.CHEVROLET_VOLT_2019, CAR.CHEVROLET_BOLT_EUV}
 
 # We're integrated at the camera with VOACC on these cars (instead of ASCM w/ OBD-II harness)
-CAMERA_ACC_CAR = {CAR.CHEVROLET_BOLT_EUV, CAR.CHEVROLET_SILVERADO, CAR.CHEVROLET_EQUINOX, CAR.CHEVROLET_TRAILBLAZER}
+CAMERA_ACC_CAR = {CAR.CHEVROLET_BOLT_EUV, CAR.CHEVROLET_SILVERADO, CAR.CHEVROLET_EQUINOX, CAR.CHEVROLET_TRAILBLAZER, CAR.GMC_YUKON}
+
+# Alt ASCMActiveCruiseControlStatus
+ALT_ACCS = {CAR.GMC_YUKON}
```

Comment:
Can we detect this?

Reply:
I think this would do it in interface.py:
`ret.networkLocation == NetworkLocation.fwdCamera and 0x370 not in fingerprint[CanBus.POWERTRAIN]`

What would be the best way to pass that to carstate?

Reply:
You can utilize `CP.flags`

Reply:
Longitudinal control is not the same, `ALT_ACCS` does not have OP long. I think these cars have a self contained radar module that send different gas/brake/status messages.